### DISCUSSION TITLE
DotReporter: extension in suite config

### DISCUSSION THREAD

**Discussion by dzoeteman** [Codeception/Codeception#6190] [Ideas]
If I add the DotReporter extension to a specific suite config, it seems to print from both reporters in the console (get both the test names from the default reporter and the dots from DotReporter). If I add it in the main codeception config, it does work correctly and only prints the DotReporter output.

Example in unit.suite.yml:
```yaml
extensions:
    enabled: [Codeception\Extension\DotReporter]
```

Could this be resolved? It's useful to have the DotReporter only for unit tests, as there are usually many more unit tests than acceptance tests.

**Comment by DavertMik** [comment]
Good idea. But I'm not so sure how it could be implemented yet
